src/pool/srv_pool.c

```diff
@@ -4166,7 +4211,11 @@ pool_svc_update_map_internal(struct pool_svc *svc, unsigned int opc,
 		D_ASSERT(tgts->pti_ids == NULL);
 		D_ASSERT(tgt_addrs != NULL);
 		D_ASSERT(inval_tgt_addrs != NULL);
-		rc = pool_find_all_targets_by_addr(map, tgt_addrs, tgts,
+		if (!is_command && opc == POOL_REINT)
+			rc = pool_find_all_targets_without_flags_by_addr(map, tgt_addrs, tgts,
+						   inval_tgt_addrs, PO_COMPF_COMMAND);
```

Comment:
Hmm, the only thing this function does is to check whether tgt_addrs have any invalid ranks/tgts, but it does nothing about it, so you skip it.

Hmm, probably this tgt_addrs -> tgts conversion should be moved outside this function, and it only take pool_target_id_list as the input parameter. @liw

Reply:
@wangdi1, you know this addr-versus-id thing better than I do. I have no strong opinion, as long as things don't break.